Generator/Templates/module_type_registration.sbntxt

```diff
@@ -7,16 +9,40 @@ namespace {{ namespace.name }}
     {
         static partial void RegisterTypes()
         {
+            {{ if namespace.name != "GLib" # TODO: Temporary until we merge GLib + GObject }}
             {{ for $class in classes}}
             try
             {
                 {{ $class | write_type_registration }}
             }
             catch (Exception e)
             {
-                Console.WriteLine($"Could not register type '{{ $class.name }}': {e.Message}");
+                Console.WriteLine($"Could not register class type '{{ $class.name }}': {e.Message}");
             }
             {{ end }}
+            
+            {{ for $record in records }}
+            try
+            {
+                {{ $record | write_type_registration_record }}
+            }
+            catch (Exception e)
```

Comment:
Why do we need a catch here? If it fails the application should crash. It just maps the types in a dictionray.

Reply:
If there are records / classes / unions without a `GetGType` method we should not fail with an exception but gracefully handle this case.

Reply:
The try catch is needed because there are some GTK classes like "GtkPlug" which are not available on all operating systems. The "GtkPlug" got removed in Gtk4. which should solve this issue. So just add a TODO which states that the try catch should be removed if we switch to Gtk4.